shapely/geometry/polygon.py

```diff
@@ -263,27 +263,6 @@ def interiors(self):
             return []
         return InteriorRingSequence(self)
 
-    def __eq__(self, other):
```

Comment:
This was an oversight in https://github.com/Toblerity/Shapely/pull/1239 where I removed the other `__eq__` methods